index.html

```diff
@@ -312,6 +312,60 @@ <h1>SAVE Data (file0)</h1>
                 <label>Plot value</label>
                 <select title="Plot value" id="sav-plotvalue" class="w-full mt-1"></select>
             </fieldset>
+
+            <fieldset class="col-span-2">
+                <legend>FUN Values</legend>
+                <div class="grid-cols-2">
+                    <table>
+                        <tr>
+                            <th>Value</th>
+                            <th>Effect</th>
+                        </tr>
+                        <tr>
+                            <td>2-39</td>
+                            <td>Wrong number</td>
+                        </tr>
+                        <tr>
+                            <td>40-45</td>
+                            <td>Sans' call</td>
+                        </tr>
+                        <tr>
+                            <td>46-50</td>
+                            <td>Alpyhs' call</td>
```

Comment:
`Alphys'` or `Alphys's`, but not `Alpyhs'`.

Reply:
i can never type it correctly